staging/vhost-device-gpu/src/virt_gpu.rs

```diff
@@ -628,18 +628,22 @@ impl VirtioGpu {
     fn update_scanout_resource(
         &mut self,
         gpu_backend: &mut GpuBackend,
-        gpu_scanout: VhostUserGpuScanout,
+        gpu_scanout: &mut VhostUserGpuScanout,
         resource_id: u32,
         scanout_data: Option<VirtioScanoutBlobData>,
     ) -> VirtioGpuResult {
-        gpu_backend.set_scanout(&gpu_scanout).map_err(|e| {
+        gpu_backend.set_scanout(gpu_scanout).map_err(|e| {
             error!("Failed to set scanout from frontend: {}", e);
             ErrUnspec
         })?;
 
         // Virtio spec: "The driver can use resource_id = 0 to disable a scanout."
         if resource_id == 0 {
-            error!("NOT IMPLEMENTED: disable scanout");
+            //To disable a scanout, the dimensions width/height are set to 0.
+            if gpu_scanout.height != 0 && gpu_scanout.width != 0 {
+                gpu_scanout.height = 0;
```

Comment:
If we need to eventually store the width/height somewhere, or in general store a list of existing scanouts it could make sense to disable the scanout there somehow (remove it from a list or even set the width and height to 0 - but I don't see why).

Reply:
you already set the scanout width and height to 0 in your draft PR, we can drop this PR.